reference/generators/cmake_find_package_multi.rst

```diff
@@ -81,3 +82,18 @@ the *CMakeLists.txt*.
 Moreover, this also adjusts `CMAKE_MODULE_PATH` and `CMAKE_PREFIX_PATH` to the values declared by the package in ``cpp_info.buildirs``, so
 modules in those directories can be found.
 
+Components
+++++++++++
+
+If a recipe uses :ref:`components <package_information_components>`, the targets generated will be ``<PKG-NAME>::<COMP-NAME>`` with the following properties adjusted. Being
+``<COMP-NAME>`` the package name used in the reference (by default) or the one declared in ``cpp_info.components["comp_name"].name`` or in 
```

Comment:
Do not document the ``cpp_info.components["comp_name"].name`` if we are planning to leave only the ``names`` one.

Reply:
Are we going to remove the `name`? The `name` assign the same value to all the generators, I think it is useful.... what we want to remove is **reading the `name` value**, it makes no sense to retrieve a _name_ if not for a concrete generator

Reply:
Not what I understood, I thought that we wanted to simplify that, and make everything explicit per-generator. But I am fine with it, just lets make sure that we are all aligned.

Reply:
We need to maintain the setter for `name`. It will help to change the name of a package (i.e.: [pro-mdnsd](https://github.com/conan-io/conan-center-index/blob/master/recipes/pro-mdnsd/all/conanfile.py), it is not using this approach, but it would be much better):

```python
class Recipe
    name = "pro-mdnsd"

    def package_info:
        self.cpp_info.name = "mdnsd"
```

You really want all the generators to use name "mdnsd", but you cannot list all the generators... you don't even know all the generators.